compiler/rustc_hir_typeck/src/fn_ctxt/_impl.rs

```diff
@@ -1275,6 +1275,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
 
             fn provided_kind(
                 &mut self,
+                _preceding_args: &[ty::GenericArg<'tcx>],
```

Comment:
Could actually allow generic const generics now here, but there's probably other work required elsewhere to support it properly

Reply:
Yeah basically nothing is set up for that at all lol